### ISSUE TITLE
Message is not in dark mode

### ISSUE THREAD

**Issue by caiqueacct** [roundcube/roundcubemail#8317]
Hi

The dark mode is working for everything except for the mail reading section. Am I missing something?

Thanks
<img width="1533" alt="Screen Shot 2021-11-17 at 22 50 52" src="https://user-images.githubusercontent.com/66717385/142366628-69720315-eb5c-4eea-9593-69f8bb3e7d80.png">

**Comment by caiqueacct** [comment]
![Uploading Screen Shot 2021-11-17 at 22.55.11.png…]()

**Comment by caiqueacct** [comment]
Version 1.5
<img width="601" alt="Screen Shot 2021-11-17 at 22 55 11" src="https://user-images.githubusercontent.com/66717385/142367097-b5e85e54-6e4f-49df-a05a-4939e8ec1e0b.png">

**Comment by alecpl** [comment]
Was that an upgrade from an earlier version? The "watermark" page is probably cached by the browser.

**Comment by caiqueacct** [comment]
Yes. upgraded from 1.4.10. You're correct. I logged in from a private windows and everything looks good. Thx!

**Comment by caiqueacct** [comment]
But the content is still white in some messages
<img width="1327" alt="Screen Shot 2021-11-18 at 01 19 09" src="https://user-images.githubusercontent.com/66717385/142387005-deb87db5-4dbf-4eea-b8a2-4333e5aa3380.png">
. not sure if this is expected

**Comment by caiqueacct** [comment]
<img width="785" alt="Screen Shot 2021-11-18 at 01 21 05" src="https://user-images.githubusercontent.com/66717385/142387246-5dd11f23-96f3-46fd-ac8f-d5009535ade9.png">

**Comment by alecpl** [comment]
This is expected for html messages. The same for the HTML editor.
